docs/Makefile

```diff
@@ -28,6 +28,8 @@ clean:
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
 	make buildapi
+	rm -rf source/developers
+	cp -R developers source/
```

Comment:
we could actually just consider putting this in source.  The only reason I didn't with release notes is that it felt like they belonged higher.  But these really are docs, and could live there.

Reply:
also, if we _do_ keep the `developers` docs folder where it and just copy it when building docs, don't forget to add `docs/source/developers/` to `.gitignore`